listenbrainz_spark/config.py.sample

```diff
@@ -19,10 +19,10 @@ SPARK_RESULT_QUEUE = "spark_result"
 # calculate stats on X months data
 STATS_CALCULATION_WINDOW = 1
 
-LOG_SENTRY = {
-    'dsn':'',
-    'environment': 'development',
-}
+#LOG_SENTRY = {
+#    'dsn':'',
+#    'environment': 'development',
```

Comment:
how do we set config.py in production? In that case, we need to make sure that we set environment properly there

Reply:
We do set the environment in production to `production`.